spec/features/toc_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "Table of Contents", type: :feature, js: true do
+  describe "links in the table of contents" do
+    it "has real href links" do
+      visit "/catalog/MC221"
+
+      expect(page).to have_css("#toc")
+      expect(page).to have_css("#toc .jstree-node", wait: 10)
```

Comment:
Why the `wait`? Can't it just be `expect(page).to have_css("#toc .jstree-node")`?